@navikt/core/react/src/progress-bar/ProgressBar.tsx

```diff
@@ -0,0 +1,129 @@
+import cl from "clsx";
+import React, { HTMLAttributes, forwardRef, useEffect, useState } from "react";
+
+interface ProgressBarPropsBase
+  extends Omit<HTMLAttributes<HTMLDivElement>, "role"> {
+  /**
+   * Changes height of progressbar.
+   * @default "medium"
+   */
+  size?: "large" | "medium" | "small";
+  /**
+   * Value of progressbar.
+   */
+  value: number;
+  /**
+   * Minimum progress.
+   * @default 0
+   */
+  valueMin?: number;
+  /**
+   * Maximum progress.
+   */
+  valueMax: number;
+  /**
+   * Used to approximate a task duration in seconds.
+   * ProgressBar shows an indeterminate animation after duration is done.
+   */
+  duration?: number;
+  /**
+   * String ID of the element that labels the progress-bar.
+   * Not needed if `aria-label` is used.
+   */
+  "aria-labelledby"?: string;
+  /**
+   * String value that labels the progress-bar.
+   * Not needed if `aria-labelledby` is used.
+   */
+  "aria-label"?: string;
+}
+
+export type ProgressBarProps = ProgressBarPropsBase &
+  (
+    | {
+        "aria-labelledby": string;
+        "aria-label"?: never;
+      }
+    | {
+        "aria-label": string;
+        "aria-labelledby"?: never;
+      }
+  );
+
+/**
+ * ProgressBar
```

Comment:
Her er noen forslag til superkort beskrivelse av komponenten:
- Shows the progress of a task or process.
- Visualizes the progression of a task or process.
- Visualizes how far along the user is in a process.

Eventuelt andre kombinasjoner av show/visualize, progress/progression, task/process.